jellyfin_kodi/downloader.py

```diff
@@ -282,6 +282,17 @@ def get_query_params(params, start, count):
             query['params'] = params
 
             result = result or {'Items': []}
+
+            # Mitigates #216 till the server validates the date provided is valid
+            for count in range(0, len(result["Items"])):
```

Comment:
With the threading above, this shouldn't need to be a loop.  It will always only be processing 1 item at a time thanks to the concurrent library.